tests/step_implementers/deploy/test_argocd.py

```diff
@@ -101,7 +101,7 @@ def test_deploy_argo_login_error(self, argocd_mock, git_mock):
                     },
                     'deploy': {
                         'argocd-app-name' : '{org}-{app}-{service}-{repo_branch_name}-{environment}'.format(org=organization_name, service=service_name, app=application_name, repo_branch_name='testbranch' , environment=environment_name),
-                        'argocd-endpoint-url': 'http://{service}.{org}-{app}-{service}-{repo_branch_name}-{environment}.{domain}'.format(org=organization_name, service=service_name, app=application_name, repo_branch_name='testbranch' , environment=environment_name, domain=kube_app_domain),
```

Comment:
Curious about the deploy-endpoint-url name change (from argocd).  Now there are some deploy "results"  that are "argocd" and some "deploy".

Reply:
Need to cross reference with feature/612, if deploy is used, possibly the comments for pdoc will need to be updated.

Reply:
changing to use deploy-endpoint in feature/612

Reply:
Thoughts about the change here?  What do you think the keys should be?

Reply:
@akim309 Ashley updated these in feature/612.

Reply:
Cool just followed up with @akim309 and this is all good.